node/churner_client.go

```diff
@@ -41,7 +43,13 @@ func NewChurnerClient(churnerURL string, useSecureGrpc bool, timeout time.Durati
 	}
 }
 
-func (c *churnerClient) Churn(ctx context.Context, operatorAddress string, keyPair *core.KeyPair, quorumIDs []core.QuorumID) (*churnerpb.ChurnReply, error) {
+func (c *churnerClient) Churn(
+	ctx context.Context,
+	operatorAddress string,
+	keyPair *core.KeyPair,
```

Comment:
This one also seems unused. Weird that linter isn't catching unused variables.

Reply:
2e43636e769b61079858a316134ecdb5e508b517